hasher-matcher-actioner/.gitignore

```diff
@@ -46,4 +46,6 @@ main.py.zip
 settings.py 
 # __init__.py and one other file are committed as an example. 
 # all other files added are expected to be third party additions/custom implementations. 
-hmalib_extensions/
\ No newline at end of file
+
+# We comment out hmalib_extensions on our fork in order to manage extension code.
```

Comment:
Is commenting out the right approach here - can we just do not ignore the gifct folder i.e. something like:

hmalib_extensions/*
!hmalib_extensions/gifct